gplugins/design_recipe/DesignRecipe.py

```diff
@@ -84,9 +154,332 @@ def eval_dependencies(self, force_rerun_all=False) -> bool:
         Evaluate this `DesignRecipe`'s dependencies.
         Because `dependencies` are assumed to be independent,
         they can be evaluated in any order.
+
+        Parameters:
+            force_rerun_all: Forces design recipes to be re-evaluated
         """
         success = True
         for recipe in self.dependencies:
-            if force_rerun_all or (not recipe.is_fresh):
+            if force_rerun_all or (not recipe.is_fresh()):
                 success = success and recipe.eval(force_rerun_all)
         return success
+
+    def load_recipe_results(self):
+        """
+        Loads recipe results from pickle file into class attribute
+        """
+        self.recipe_results = self.recipe_results.get_pickle()
+
+    def save_recipe_results(self):
+        """
+        Saves recipe_results to pickle file while adding setup information.
```

Comment:
maybe something like "Save this `DesignRecipe`s results, as well as information about the setup that produced them, to a pickle file."?